XIVComboExpanded/Combos/VPR.cs

```diff
@@ -540,44 +548,44 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     return VPR.ThirdGeneration;
                 if (gauge.AnguineTribute == maxtribute - 3)
                     return VPR.FourthGeneration;
+                // Even though this deviates from the way it works in PvP, this is likely the more desired behavior to avoid accidentally using First Generation
+                return VPR.Ouroboros;
             }
 
-            // First step, decide whether or not we need to apply debuff
-            if (OriginalHook(VPR.SteelMaw) == VPR.SteelMaw)
+            switch (OriginalHook(VPR.SteelMaw))
             {
-                var noxious = FindTargetEffect(VPR.Debuffs.NoxiousGash); // TODO: Would be useful to handle the case with no target
-                if (level >= VPR.Levels.DreadMaw && (noxious is null || noxious?.RemainingTime < 12)) // 12s hopefully means we won't miss anything on a Reawaken window
-                    return VPR.DreadMaw;
+                case VPR.SteelMaw:
+                    var noxious = FindTargetEffect(VPR.Debuffs.NoxiousGash); // TODO: Would be useful to handle the case with no target
+                    if (level >= VPR.Levels.DreadMaw && (noxious is null || noxious?.RemainingTime <= 20)) // 20s determined by https://github.com/MKhayle/XIVComboExpanded/pull/353/files#r1690057020
+                        return VPR.DreadMaw;
+
+                    return VPR.SteelMaw;
+
+                case VPR.HuntersBite:
+                    var swift = FindEffect(VPR.Buffs.Swiftscaled);
+                    var instinct = FindEffect(VPR.Buffs.HuntersInstinct);
+                    if (swift is null) // I think we'd always want to prioritize swift since it speeds up the rotation
```

Comment:
You know, I was going to contest this, but it turns out that it's correct.  Samurai uses Gekko before Kasha, so I assumed the same priority would hold for Viper, damage buff before haste buff, but the official Balance opener is Swiftskin's Bite -> Dreadwinder -> Hunter's Coil, rather than the opposite.

Reply:
I am so glad we're getting a division between nerd combos and basic combos